weave-js/src/components/Panel2/PanelTable/ColumnHeader.tsx

```diff
@@ -430,8 +436,8 @@ export const ColumnHeader: React.FC<{
   const colIsSorted =
     tableState.sort.find(sort => sort.columnId === colId)?.dir != null;
 
-  const colControlsWidth =
-    20 * (1 + (colIsSorted ? 1 : 0) + (isPinned ? 1 : 0));
+  // const colControlsWidth =
```

Comment:
This appears to be used down below to pull the column name towards the right towards the controls. I commented this out because with this PR, we are making column headers either left- or right-aligned so this should no longer be needed. 

I think the correct decision is to delete this code and the below use of `colControlsWidth` but wanted to pause & see if there are any downstream consequences. Clearly the calculations here are intentional; I wanted to make sure there aren't any downstream consequences.